README.md

```diff
@@ -12,8 +13,8 @@ There are 2 versions on NPM, but only [the one in the DagreJs org](https://www.n
 
 # License
 
-Graphlib is licensed under the terms of the MIT License. See the
```

Comment:
Was this supposed to be removed? The license should still be included in the readme.

Reply:
> 

I think you're right.